src/app/state/hooks.js

```diff
@@ -149,15 +168,76 @@ export function useCoordination(parameters, coordinationScopes) {
     }));
   }, shallow);
 
-  const setters = fromEntries(parameters.map((parameter) => {
+  const setters = useMemo(() => fromEntries(parameters.map((parameter) => {
     const setterName = `set${capitalize(parameter)}`;
     const setterFunc = value => setCoordinationValue({
       parameter,
       scope: coordinationScopes[parameter],
       value,
     });
     return [setterName, setterFunc];
-  }));
+  })), [parameters, coordinationScopes]);
```

Comment:
```suggestion
  // eslint-disable-next-line react-hooks/exhaustive-deps
  })), [parameters, coordinationScopes]);
```
Can you add these ignore statements to resolve the warnings about `setCoordinationValue`